src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,20 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+// Add the required types and props
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <>
```

Comment:
You don't need fragment here, since there is no elements that needs not be grouped